assets/app/scripts/controllers/catalog/images.js

```diff
@@ -12,9 +12,26 @@ angular.module('openshiftConsole')
     $scope.projectImageRepos = {};
     $scope.openshiftImageRepos = {};
     $scope.builders = [];
-    $scope.images = [];
+    $scope.nonBuilders = [];
     $scope.sourceURL = $routeParams.builderfor;
 
+    var isBuilder = function(imageRepo) {
```

Comment:
Should the particular status.tag we're checking be an input here, rather than checking annotations.tags on all spec.tags to determine if a particular status.tags[x] is a builder image?